custom/ewsghana/models.py

```diff
@@ -68,8 +69,6 @@ def update_toggle(self):
             STOCK_AND_RECEIPT_SMS_HANDLER.set(self.domain, True, NAMESPACE_DOMAIN)
 
 
-class AlertsSent(models.Model):
-    create_date = models.DateTimeField(editable=False)
-    alert_type = models.TextField()
-    alert_text = models.TextField()
-    supply_point_id = models.CharField(max_length=10)
+class FacilityInCharge(models.Model):
```

Comment:
No, it's not for efficiency. Now when we assign incharge role to particular user, he becomes in charge in location that he is assigned. This PR changes this behavior. Role in charge doesn't mean that user is in charge for some location but it means that he can be selected as in charge for some location. I hope that now is clear. 

Here is the spec written by Mike:
http://manage.dimagi.com/default.asp?176370#980093